example/datatypes.card.js

```diff
@@ -72,6 +72,11 @@ var ListToggle = React.createClass({
   }
 });
 
+devcard('Set',
```

Comment:
I think you need to add the babel runtime for older browsers yes. Don't remember if webpack doesn't inject some kind of polyfill for you however.